src/Nethermind/Nethermind.JsonRpc/Modules/DebugModule/JavaScriptObjectConverter.cs

```diff
@@ -0,0 +1,66 @@
+// SPDX-FileCopyrightText: 2023 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+using System.Collections.Generic;
+using System.Numerics;
+using Microsoft.ClearScript;
+using Microsoft.ClearScript.JavaScript;
+using Newtonsoft.Json;
+
+namespace Nethermind.JsonRpc.Modules.DebugModule;
+
+public class JavaScriptObjectConverter : JsonConverter
```

Comment:
Could we add tests for this converter?

Reply:
I think it is already somewhat indirectly tested, but can add more explicit tests

Reply:
yeah it is already well tested in serialization of js tests